specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2023-07-01/imds.json

```diff
@@ -24,6 +24,34 @@
     }
   },
   "paths": {
+    "/versions": {
+      "get": {
+        "operationId": "GetVersions",
+        "description": "Get supported API versions.",
+        "responses": {
+          "200": {
+            "description": "Successful operation",
+            "schema": {
```

Comment:
1. You should not return an array as the schema. You cannot change it later. Instead, you should return an object with an array. We generally recommend something like `{"value":[...]}`. See https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#collections
2. This doesn't match your `GetVersions.json` sample which returns an object with a top-level array named `apiVersions` (we recommend `value`, but not a hard requirement).

Reply:
Fixed the schema, could you take a look again?

Reply:
@heaths @xxxxiehf This isn't a change we could make in this context. The versions endpoint has existed since 2020-10-01 and is carried forward to each new API version. It looks like the issue is it's missing from the API spec by error, which is why it looks like a new feature to you.

Reply:
@LittleBoxOfSunshine I didn't change the schema, actually the schema I added here original is wrong and didn't match the one in `GetVersions.json` file. So I was fixing it.